docs/get-started.md

```diff
@@ -147,6 +147,24 @@ gittuf allows for verifying rules for Git references and files.
 $ gittuf verify-ref --verbose main
 ```
 
+## Verify gittuf itself
+
+You can also verify the state of the gittuf source code repository with gittuf
+itself. At the moment, you'll need to pull down gittuf's metadata manually.
+
+You can use gittuf to clone the repository and gittuf metadata.
+
+```bash
+$ gittuf clone https://github.com/gittuf/gittuf
+```
+
+gittuf performs verification automatically with the above command, but you can
+also verify for yourself again.
+
+```bash
+$ gittuf verify-ref --verbose v0.4.0
```

Comment:
the clone will fetch the main branch, not this tag, so they'd have to fetch that separately.